app/views/users/show.html.erb

```diff
@@ -1,163 +1,239 @@
 <% content_for :title, "User #{@user.rtl_safe_username}" %>
 
+<h1><span dir="ltr"><%= @user.rtl_safe_username %></span></h1>
+
 <% if @user.community_user.suspended? %>
-<div class="notice is-danger">
-<% if @user.community_user.suspension_public_comment.nil? %>
-  <p>This user has been <strong>temporarily suspended</strong>.
-<% else %>
-  <p>This user has been <strong>temporarily suspended</strong> <%= @user.community_user.suspension_public_comment %>.
-<% end %>
+  <div class="notice is-danger h-m-b-4">
+  <% if @user.community_user.suspension_public_comment.nil? %>
+    <p>This user has been <strong>temporarily suspended</strong>.
+  <% else %>
+    <p>This user has been <strong>temporarily suspended</strong> <%= @user.community_user.suspension_public_comment %>.
+  <% end %>
 
-  The suspension ends <span title="<%= @user.community_user.suspension_end.iso8601 %>">in <%= time_ago_in_words(@user.community_user.suspension_end) %></span>.</p>
-</div>
+    The suspension ends <span title="<%= @user.community_user.suspension_end.iso8601 %>">in <%= time_ago_in_words(@user.community_user.suspension_end) %></span>.</p>
+  </div>
 <% end %>
 
-<%= render 'tabs', user: @user %>
+<div class="tabs is-classic-tabs">
+  <%= link_to user_path(@user), class: "tabs--item is-active" do %>
+      <i class="fas fa-user"></i> Profile
+    <% end %>
+  <%= link_to user_activity_path(@user), class: "tabs--item" do %>
+    Activity
+  <% end %>
+  <% if @user == current_user %>
+    <%= link_to edit_user_registration_path, class: "tabs--item" do %>
+      Account
+    <% end %>
+    <%= link_to edit_user_profile_path, class: "tabs--item" do %>
+      Edit
+    <% end %>
+    <%= link_to subscriptions_path, class: "tabs--item" do %>
+      Subscriptions
+    <% end %>
+  <% end %>
+</div>
```

Comment:
**question**: why is this expanded from the `render 'tabs'`? If there are changes needed, we should put them in the tabs view; we get out of sync quickly otherwise.